app/javascript/utils.js

```diff
@@ -64,3 +64,98 @@ function sortArrayByKey(array, key) {
 }
 
 export { slugify, shuffleArray, getName, sortArrayByKey };
+
+export const circles = [
```

Comment:
In `app/javascript/components/party/index.js` we have this. Can we avoid duplicationg?

Reply:
Algo a refactorizar no futuro. Não quero importar do ficheiro do componente, mas também não quero tocar em algo que teria estar responsável de garantir que ficava tudo ok. Então, melhor duplicar por estas duas semanas e garantir que nada parte.

Aliás, neste momento, não são o mesmo array. Tive de capitalizar "Fora da europa" para bater certo com o que vem de BE. Acho que fica a discussão para depois, podemos meter typescript... ou não. Por um lado, melhor para garantir a tipificação, por outro pode ser um entrava a FE que não conheçam TS. 💭